test/test_tomopy/test_misc/test_corr.py

```diff
@@ -66,6 +66,21 @@ def test_gaussian_filter(self):
     def test_median_filter(self):
         loop_dim(median_filter, read_file('cube.npy'))
 
+    def test_median_filter_nonfinite(self):
+        data_org = np.ones(shape=(100, 100, 100))
+
+        for i in range(50):
+            x = np.random.randint(0, 100)
+            y = np.random.randint(0, 100)
+            z = np.random.randint(0, 100)
+            data_org[z, x, y] = np.inf
```

Comment:
```suggestion
        for i in range(50):
            x = np.random.randint(0, 100)
            y = np.random.randint(0, 100)
            z = np.random.randint(0, 100)
            data_org[z, x, y] = np.inf
            x = np.random.randint(0, 100)
            y = np.random.randint(0, 100)
            z = np.random.randint(0, 100)
            data_org[z, x, y] = -np.inf
            x = np.random.randint(0, 100)
            y = np.random.randint(0, 100)
            z = np.random.randint(0, 100)
            data_org[z, x, y] = np.nan
```
We need to test all types of nonfinite values.